### DISCUSSION TITLE
WLAN + LAN gleichzeitig in verschiedenen Netzen nutzen

### DISCUSSION THREAD

**Discussion by IT-VBFK** [OpenCCU/OpenCCU#1012] [🇩🇪 Ideen]
Hallo,

Evtl. wäre es interessant, LAN und WLAN gleichzeitig zu nutzen; dies aber nicht im selben Netz, sondern zB das WLAN in einem vom Internet getrennten Netz und das LAN im normalen Netz mit Internet. 

Grund dafür könnte sein: man will andere smart home Gadgets via MQTT oder sonstigen Protokollen anbinden (zB via RedMatic o.ä.). Allerdings möchte man diesen Gadgets nicht das gesamte Netz "überlassen", sondern diese in logische, getrennte Netze geben.

Wollte mal hier nachfragen, wie man zu diesem Thema stehen würde und ob das hier eilt. eine Chance hat.

**Comment by jens-maus** [comment]
Auch wenn ich den Wunsch von Netzwerkseite her schon verstehen kann so befürchte ich, das so eine Funktionalität in der WebUI zu implementieren zu viel Komplexität reinbringen würde.

Darüber hinaus müsste das aber schon jetzt prinzipiell über Shell-skripting individuell umsetzen sein. D.h. wenn jemand solch ein spezielles Setup braucht dann sollte sich das jetzt schon - wenn auch nur via kommandozeile+scripting - umsetzen lassen.

**Comment by IT-VBFK** [comment]
Ok, ich meinte aber jetzt unabhängig von der WebUI. Im Grunde würde es schon reichen, wenn man die Anleitung bezgl. WLAN Einrichtung 1:1 befolgen kann, aber dann nicht das LAN-Kabel abstecken muss, um das WLAN-Modul nutzen zu können.

Beispiel:
vorhanden Netzwerk A (mit Internetzugriff über LAN) und Netzwerk B (ohne Internetzugriff (bzw. VLAN) über WLAN)

Das hat ja erstmal nichts mit der WebUI zu tun, sondern muss ja auf Betriebsystem-Ebene passieren.

**Reply by jens-maus** [reply]
Das macht es dann aber schwierig für otto-normal-verbraucher das in einer art und weise umzusetzen, das die das noch unterm strich gemanaged bekommen. Genau deshalb ist es aktuell so umgesetzt wie von dir beschrieben.

**Reply by IT-VBFK** [reply]
ok, alles klar

bezgl. selber via command line + scripting: hast du da irgendeinen Ansatz wie man das angehen könnte?

**Comment by XiaoJun9** [comment]
Hallo,
ich habe dieselbe Frage: Ich würde gern das wlan0 vom Rapi separat als accesspoint für z.B. mqtt über wlan, Sensoren und so konfigurieren. 
Es gelang mir zwar, das wlan zu aktivieren, scan funktioniert. Ich bin mir aber nicht sicher, was ich tun muß, um das als AP dann auch zugänglich zu haben. Ich dachte an eine config mit Mosquitto und Redmatik...
Die allgemeinen, auf Buster gerichteten Anweisung hängen z.T. an den schreibgeschützen dirs in Rspberrymatic. Ich will aber das System auch nicht unnötig "aufbrechen".
Wo finde ich da Infos?
